R/hexbin.R

```diff
@@ -31,6 +31,8 @@ hexBinSummarise <- function(x, y, z, binwidth, fun = mean, fun.args = list(), dr
 
   # Convert to data frame
   out <- as.data.frame(hexbin::hcell2xy(hb))
+  out$count = as.vector(hb@count)
```

Comment:
Also I'm not sure that the code should go here - it's only needed for hexbin, not the other summary functions.